docs/content/upgrade-guides/admin/1-0-0.md

```diff
@@ -0,0 +1,40 @@
+# Medusa Admin: 1.0.0
+
+Medusa Admin has been moved from a standalone repository to our mono-repository to centralise projects and ship it as an NPM package.
+
+## Overview
+
+The NPM package name is `@medusajs/admin-ui` and in our core repository the UI can be found within `packages/admin-ui`. 
```

Comment:
Some comment as the previous suggestion. If it's not important to highlight details regarding the repository, then I think we can just say that admin is now composed of two packages `@medusajs/admin` and `@medusajs/admin-ui`, and explaining generally what each is.

If it's important to highlight the repository, then we should use the same terms when referring to it. Before you mentioned it as "mono-repository" but now it's "core repository"